l10n/en-US/viewer.ftl

```diff
@@ -446,7 +446,7 @@ pdfjs-editor-new-alt-text-error-close-button = Close
 #   $totalSize (Number) - the total size (in MB) of the AI model.
 #   $downloadedSize (Number) - the downloaded size (in MB) of the AI model.
 #   $percent (Number) - the percentage of the downloaded size.
-pdfjs-editor-new-alt-text-ai-model-downloading-progress =
+pdfjs-editor-new-alt-text-ai-model-downloading-progress = Downloading alt text AI model ({ $downloadedSize } of { $totalSize } MB)
```

Comment:
https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html#general-guidelines

Changes to existing string require new IDs.

Reply:
Actually, this hasn't been exported to `mozilla-central` yet, so we're good (as long as we don't push the previous version). Can we guarantee that?

Reply:
The string doesn't exist yet in m-c, so it should be ok like this, right ?

Reply:
I plan to do a release today.